### PR TITLE
Add in a read-only prober across Rekor and Fulcio API endpoints

### PR DESCRIPTION
**Pull request by priyawadhwa**
This prober performs all read requests across endpoints and exports data from prometheus. we can run it in the staging and production clusters to track metrics and set alerts.

### CODE REVIEW

File: config/prober/200-deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      labels:
+        app: sigstore-prober
+      annotations:
+        prometheus.io/scrape: "true"
+        prometheus.io/path: /metrics
+        prometheus.io/port: "8080"
+    spec:
+      containers:
+      - name: sigstore-prober
+        image: ko://github.com/sigstore/scaffolding/cmd/prober
```
Comment (cpanato) [inline config/prober/200-deployment.yaml:25]: maybe you can add some fine grain definition in the file https://github.com/sigstore/scaffolding/blob/main/.ko.yaml similar we have for the others

also do we need to release this image as well? or we build this adhoc?

Comment (priyawadhwa) [reply config/prober/200-deployment.yaml:25]: thanks @cpanato, added in the definition!

so for this i was thinking we could release the image and then maybe package it into a helm chart so I could install it in staging and prod. WDYT?

Comment (cpanato) [reply config/prober/200-deployment.yaml:25]: agree! then for that we need to add a rule in the make file and glue in the release, i can do that if you want as a follow up

Comment (priyawadhwa) [reply config/prober/200-deployment.yaml:25]: that would be awesome, thank you :D

**Review by cpanato — APPROVED** [review]

**Pull request merged**